src/app/components/TextField/styles.module.scss

```diff
@@ -2,8 +2,8 @@
 
 .textfield-container {
   .title {
-    display: inline-block;
     color: $black;
+    display: inline-block;
```

Comment:
we avoid setting `block` and inline-block`. if you need a text to use full line use flex